### PR TITLE
Added project euler problem7 solution in python

### PR DESCRIPTION
**Pull request by parulagg27**
**Fixes issue:**

**Changes:**

### CODE REVIEW

File: code/online_challenges/src/project_euler/problem_007/problem_007.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+	list_of_primes = []
+	
+	while (len(list_of_primes) < n):
+		if all (x % prime for prime in list_of_primes):
+			list_of_primes.append(x)
+		x += 1
+
+	print (list_of_primes[-1])
+
+
```
Comment (mansiag) [inline code/online_challenges/src/project_euler/problem_007/problem_007.py]: Remove this extra newline

Comment (parulagg27) [reply code/online_challenges/src/project_euler/problem_007/problem_007.py]: I don't think this should be removed. According to PEP 8 style guide, we need to follow for python code in this repository, there should be 2 blank lines after class or function definition.

Comment (arnavb) [reply code/online_challenges/src/project_euler/problem_007/problem_007.py]: @parulagg27 Looks like you're right (You also need this at the top of the file). However, I haven't really seen other files in Python follow this convention, so I would just remove the line.

Comment (parulagg27) [reply code/online_challenges/src/project_euler/problem_007/problem_007.py]: Oh, alright.

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

positive_opengenus_cosmos

**Pull request merged**